iOS-sample-app/Showcases/Transactions/TransactionsViewController.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import ContentSquare
+
+struct MyItem
+{
+    private(set) var myTitle: String
+    private(set) var myPrice: Float
+    private(set) var myCurrency: Currency
+
+    init(title: String, price: Float, currency: Currency)
+    {
+        myTitle = title
+        myPrice = price
+        myCurrency = currency
+    }
+}
+
+// This class showcases how to track purchases made by an user with CustomerTransactions
+class TransactionsViewController: UIViewController {
+
+    private var itemsInCart = [MyItem]()
+    private var selectedCurrency : Currency = .eur
+    private var itemCount = 0
+
+    override func loadView()
+    {
+        super.loadView()
+
```

Comment:
Could you add a little explanatory text on the screen itself? For instance "This screen showcases how to send transactions. Add items to the cart, then when you validate the cart a transaction will be sent."

I'd like people running the app to understand a screen when they get to it.

Reply:
@XavierC-cs  I believe @teogeos meant to put the text in the UI so the user of the sample can see it.